modules/formulize/include/functions.php

```diff
@@ -2737,6 +2737,17 @@ function formulize_text_to_hyperlink($text, $textWidth) {
     return str_replace("<a ", "<a target='_blank' ", $text);
 }
 
+/**
+ * Strip <?php out of the beginning of a string if it is present (checks the first five chars)
+ * @param string $string The string you want to check and strip from
+ * @return string The string, potentially modified with <?php gone if it was found as the first five chars
+ */
+function removeOpeningPHPTag($string) {
```

Comment:
This may be handled elsewhere by the core system but what protection do we have to ensure that no spaces, newline, or invisible characters are placed at the beginning these strings? If any of those characters end up here without a trim in place this function will not strip the PHP tag out.